packages/ui/src/styles/base.scss

```diff
@@ -0,0 +1,3 @@
+@import "../base/layout";
```

Comment:
kinda nit: I wonder if the `base` folder should be inside the `style` folder.

Reply:
Me too! I was wondering if we need this `base` folder. 

btw, probably we won't need this `layout.scss`, since it's related to the page's styles (probably move it to store _in the future_)

Reply:
nop, because styles is the folder that the store can import then. In the future, faststore/ui will have just scss files, not necessarily only in the style folder.